uniqueid.r2py

```diff
@@ -7,20 +7,12 @@ Start date: November 11th, 2008
 
 This is a really, really simple module, only broken out to avoid duplicating 
 functionality.
-
-NOTE: This will give unique ids PER FILE.   If you have multiple python 
-modules that include this, they will have the potential to generate the
-same ID.
-
 """
 
-# This is a list to prevent using part of the user's mycontext dict
-# We use getruntime() instead of a list starting with 0, as this 
-# library may get imported multiple times. 
-# See ticket #1319 and #1318 for more details.
+# Use this list as module-level storage. (This avoids cluttering the
+# programmer's `mycontext` dict.)
 
-current_time = getruntime()
-uniqueid_idlist = [int((current_time - int(current_time))  * 2**32)]
+uniqueid_idlist = [0]
```

Comment:
Why exactly does this have to be a list? It will always only hold one element, right?